demo-compose.yml

```
@@ -67,12 +76,13 @@ services:
   web-ui:
     command: ["uwsgi /opt/invenio/var/instance/uwsgi_ui.ini"]
     pull_policy: always
-    image: ghcr.io/sharedrdm/instance:main-mug
-    env_file: .env.temp
+    image: ghcr.io/sharedrdm/instance:v0.13.0-mug
```

```suggestion
    image: ghcr.io/sharedrdm/instance:${RELEASE_TAG}
```
add RELEASE_TAG inside the `.env`